GettingStarted.md

```diff
@@ -1,12 +1,176 @@
 # Getting Started with Localytics in Xamarin
 ---
-This Xamarin Component simplifies the process of integrating Localytics into Xamarin iOS, Android and Forms projects.
+This Xamarin Nuget simplifies the process of integrating Localytics into Xamarin iOS, Android and Forms projects.
 
 The steps to integrate Localytics with this component mirrors the native Localytics SDK, so the following docs for the respective platforms maybe helpful:
 
 * [iOS Localytics Docs](http://docs.localytics.com/dev/ios.html)
 * [Android Localytics Docs](http://docs.localytics.com/dev/android.html)
 
+
+## Getting Started
+
+### Install the SDK
+1. Download the latest version of the Nuget package from [here](https://downloads.localytics.com/SDKs/Xamarin/Localytics-Xamarin-Latest.zip).
+2. Add a Nuget Package source in Visual Studio Preferences called localytics that points to the folder with the downloaded package.
+3. In Projects where you want to use Localytic SDK 
+4. Integrating to each of your projects
```

Comment:
This isn't super clear to me - but I'm not really a Xamarin developer.  As a thought, some screenshots may help here.